Tools/Scripts/libraries/webkitscmpy/webkitscmpy/remote/scm.py

```diff
@@ -82,6 +85,25 @@ def from_url(cls, url, contributors=None, classifier=None):
 
         raise OSError("'{}' is not a known SCM server".format(url))
 
+    @classmethod
+    def insert_diff_comments(cls, generator, comments=None):
+        file = None
+        count = 0
+        comments = comments or dict()
+        for line in generator():
+            if line.startswith('+++ b/'):
+                file = line.split('/', 1)[-1]
+                count = -1
+            yield line
+            comments_on = comments.get(file, {}).get(None if count < 0 else count, [])
+            if comments_on:
+                yield '>>>>'
+                for comment in comments_on:
+                    for line in comment.splitlines():
+                        yield line
+                yield '<<<<'
+            count += 1
```

Comment:
We can certainly play around with the formatting, but I wonder if, instead of interspersing comments with what look like conflict markers, it would be better to indenting all the code rightward one level, and leave comments unindented? That ought to help them visually stand out, without adding line noise that would confuse a screen reader.

Reply:
The good news is that this is indeed shared code, I too, am unsure that `<<<<` is the right approach.

One of the challenges here is that we're not actually interested in comments "visually" standing out, we're interested in them being easy for screen readers to parse. I did consult folks using screen readers, and `<<<<` was the implementation we settled on. I'm not certain if indenting all the code is better or worse for screen reader users, I intend to follow up with screen reader users once we have a PR which can download a diff with comments.